source/Planet.cpp

```diff
@@ -294,17 +298,26 @@ void Planet::Load(const DataNode &node, Set<Wormhole> &wormholes)
 // we must auto generate one if we detect such legacy wormhole.
 void Planet::FinishLoading(Set<Wormhole> &wormholes)
```

Comment:
Why is this being changed?

Reply:
I added defaulting an undefined display name to the true name, and regroupped the ifs to avoid duplicates.